src/sphinx_book_theme/assets/styles/base/_base.scss

```diff
@@ -2,6 +2,15 @@
 /*********************************************
 * Utility classes used in a few places
 *********************************************/
+// For the helper pixel that we can watch to decide whether we'vez
```

Comment:
we'vez?

Reply:
We'vez is the new warez 😄

Good catch lol